packages/react-components/lib/tasks/task-table-datagrid.tsx

```diff
@@ -251,8 +276,30 @@ export function TaskDataGridTable({
       field: 'status',
       headerName: 'State',
       editable: false,
-      valueGetter: (params: GridValueGetterParams) =>
-        params.row.status ? params.row.status : 'unknown',
+      renderCell: (cellValues) => {
+        const statusString = cellValues.row.status ? cellValues.row.status : 'unknown';
+        if (dismissStaleTasks && isTaskOutdated(cellValues.row)) {
+          return (
+            <Tooltip
+              title={
+                <React.Fragment>
+                  <Typography>
+                    Finish time is in the past, but task is still queued or underway.
+                  </Typography>
+                  <Typography>
+                    The task may have been interrupted or stalled during the execution.
+                  </Typography>
+                  <Typography>Last status update: {statusString}</Typography>
+                </React.Fragment>
```

Comment:
The "stale" state is not an official rmf state? Instead of hiding the last known state in a tooltip, suggest to format it together. e.g. `Underway (stale)`. This prevent a out of place state from showing when sorting/filtering.

Reply:
Fair enough, those are good points. Per discussion, I will be making this feature enabled by default, and also removing the configuration part.